README.md

```diff
@@ -0,0 +1,13 @@
+# Dune Sync V2
+
+
+
+
+## Local Development
+
+Fill out the empty fields in [Sample Env](.env.sample) (`DUNE_API_KEY` and `QUERY_ID`)
```

Comment:
This will change once we parse config files (i.e. query id wont live in the env). Make sure to update the readme when ready.

Reply:
https://github.com/bh2smith/dune-sync/issues/5